src/dune_rules/cc_rules.ml

```diff
@@ -35,8 +33,8 @@ let rules ~sctx ~dir =
        let open Command.Args in
        S
          [ (match Ocaml_config.ccomp_type ocfg with
-            | Msvc -> A "/EP"
-            | Other _ -> As [ "-E"; "-P" ])
+            | Msvc -> As [ "/nologo"; "/EP" ]
```

Comment:
I also noticed we are now passing `/nologo` to msvc. Looking at the documentation this appears to be harmless, but it might be worth pointing out to others if they ever come back to this PR.

Reply:
This is the content of the commit message:

> Disable MSVC copyright output when discovering the cc_vendor
>
> MSVC has an annoying ‘feature’: it prints a copyright statement
everytime it's called. Fortunately, it can be silenced by passing the
`/nologo` argument. MSVC always prints the name of the file it's
compiling.